src/main/lib/files.js

```diff
@@ -86,9 +86,17 @@ export function loadFile(filePath, win, isNew = false) {
   if (!win) {
     win = getMainWindow();
   }
-  if (win) {
+
+  // If we have found a window at this point
+  if (win !== null) {
```

Comment:
What about joining this to the previous if?
```javascript
   if (!win) {
      win = getMainWindow();
    } else {
        win.webContents.send('load-archive', payload);
        return;
    }
```

Reply:
Oh, actually I see why.. it occurs after. Would it be safer to just check truthiness? `if (win) {`?